### PR TITLE
neo: new recipe

### PR DESCRIPTION
**Pull request by hoanga**
the following adds the neo program as a port. sample run below:

```
> uname -a
Haiku shredder 1 hrev56137 May 27 2022 06:24:51 x86_64 x86_64 Haiku



> neo --help
Usage: neo [OPTIONS]

Simulate the digital rain from "The Matrix"

Options:
  -a, --async            asynchronous scroll speed
  -b, --bold=NUM         control character boldness
  -C, --colorfile=FILE   read the colors from a file
  -c, --color=COLOR      select the foreground text color
  -D, --defaultbg        use the default terminal background color
  -d, --density=NUM      set the density of droplets
  -F, --fullwidth        use two columns per character
  -f, --fps=NUM          set the frames per second target/limit
  -G, --glitchpct=NUM    set the percentage of screen chars that glitch
  -g, --glitchms=NUM1,2  control how often characters glitch
  -h, --help             show this help message
  -l, --lingerms=NUM1,2  control how long characters linger after scrolling
  -M, --shadingmode=NUM  set the shading mode
  -m, --message=STR      display a message
  -p, --profile          enable profiling mode
  -r, --rippct=NUM       set the percentage of droplets that die early
  -S, --speed=NUM        set the scroll speed in chars per second
  -s, --screensaver      exit on the first key press
  -V, --version          print the version
      --chars=NUM1,2     use a range of unicode chars
      --charset=STR      set the character set
      --colormode=NUM    set the color mode
      --maxdpc=NUM       set the maximum droplets per column
      --noglitch         disable character glitching
      --shortpct=NUM     set the percentage of shortened droplets

See the manual page for more info: man neo
```

### CODE REVIEW

File: app-misc/neo/neo-0.6.1.recipe [hunk 1]
```diff
@@ -0,0 +1,44 @@
+SUMMARY="Recreates the digital rain effect"
+DESCRIPTION="neo recreates the digital rain effect \
+from \"The Matrix\". Streams of random characters will \
+endlessly scroll down your terminal screen."
+HOMEPAGE="https://github.com/visit1985/neo"
```
Comment (humdingerb) [inline app-misc/neo/neo-0.6.1.recipe:5]: Wrong URL: https://github.com/st3w/neo :)

File: app-misc/neo/neo-0.6.1.recipe [hunk 2]
```diff
@@ -0,0 +1,44 @@
+SUMMARY="Recreates the digital rain effect"
```
Comment (humdingerb) [inline app-misc/neo/neo-0.6.1.recipe:1]: Maybe a bit more recognizable:
> Recreates Matrix' falling characters in Terminal

File: app-misc/neo/neo-0.6.1.recipe [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+BUILD()
+{
+	./configure --prefix=$prefix --mandir=$manDir
+	make
+}
+
+INSTALL()
+{
+
```
Comment (humdingerb) [inline app-misc/neo/neo-0.6.1.recipe:42]: stray emtpy line

**Review by humdingerb — DISMISSED** [review]

File: app-misc/neo/neo-0.6.1.recipe (hunk 1 above)
Comment (hoanga) [reply app-misc/neo/neo-0.6.1.recipe]: updated, thanks!

File: app-misc/neo/neo-0.6.1.recipe (hunk 2 above)
Comment (hoanga) [reply app-misc/neo/neo-0.6.1.recipe]: good point, updated

File: app-misc/neo/neo-0.6.1.recipe (hunk 3 above)
Comment (hoanga) [reply app-misc/neo/neo-0.6.1.recipe]: updated

**Pull request merged**